integrations/docker/images/chip-cirque-device-base/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+SOURCE=${BASH_SOURCE[0]}
+SOURCE_DIR=$(cd "$(dirname "$SOURCE")" >/dev/null 2>&1 && pwd)
+REPO_DIR="$SOURCE_DIR/../../../../"
+
+# The image build will clone its own ot-br-posix checkout due to limitations of git submodule.
+# Using the same ot-br-posix version as chip
+OT_BR_POSIX=$REPO_DIR/third_party/ot-br-posix/repo
+OT_BR_POSIX_CHECKOUT=$(cd "$OT_BR_POSIX" && git log -n 1 --format='%H')
+
+docker build -t chip-cirque-device-base -f "$SOURCE_DIR"/Dockerfile --build-arg OT_BR_POSIX_CHECKOUT="$OT_BR_POSIX_CHECKOUT" "$SOURCE_DIR"
```

Comment:
The ot-br-posix checkout is set here when building with build.sh

Reply:
does our third_party repo track master or is it fixed to a specific hash?

I see in .gitmodules:

```
[submodule "ot-br-posix"]
        path = third_party/ot-br-posix/repo
        url = https://github.com/openthread/ot-br-posix.git
        branch = master
```

which tells me that I always get the latest master build in third_party. This leads me to believe that if I do 'docker build' today and 'docker build' two weeks from now on the same CHIP sources I may get a different result. This makes tracking down CI failures a lot harder.

Reply:
is something that @mspang might have an answer for.  our trees

Reply:
We track a specific version. You want this instead:

`git rev-parse :third_party/cirque/repo`

`branch` instructs the update script about what to update to. Ideally we automate this and run it ~daily.